.golangci.yml

```diff
@@ -23,6 +23,15 @@ issues:
       text: "SA1019: codec.BasicHandle is deprecated"
     # This is set to false to disregard the default EXC0011 exclusion
     # which shadows all the ST* checks. Refer https://github.com/golangci/golangci-lint/issues/2281
+    - linters:
+        - govet
+      text: "printf: non-constant format string"
+    - linters:
+        - staticcheck
```

Comment:
These lint errors are showing up due to the new `golangci-lint version 1.60.3` upgrade. We can fix these in the future.

Reply:
In my previous experience, whoever's doing the bump does the work to munge these as they come up.

Reply:
Addressed